### ISSUE TITLE
weecloud fail on ssl.weechat relay.

### ISSUE THREAD

**Issue by Cubox** [eirikb/weecloud#13]
Hi,

I have created a relay with /relay add ssl.weechat port.
The Weechat Android application is working well with it, but weecloud is making this error:

```
relay: la poignée de main (handshake) de TLS a échoué pour le client 5/ssl.weechat/::ffff:165.225.134.101: erreur -15 Un paquet TLS incorrect a été reçu.
```

Local weecloud is worse, gives me:

```
Wed Apr 03 2013 22:57:43 GMT+0200 (CEST) '10.8.0.2'

events.js:72
        throw er; // Unhandled 'error' event
              ^
Error: read ECONNRESET
    at errnoException (net.js:878:11)
    at TCP.onread (net.js:539:19)
```

**Comment by eirikb** [comment]
I haven't looked into support of ssl yet, but I wonder if this should perhaps be a issue for [weechat.js](https://github.com/eirikb/weechat.js) instead?

**Comment by Cubox** [comment]
I don't know. I tested without ssl, and on branch dev and master, still crashing. Will get in touch with you on IRC to see if it's not specific to me.

**Comment by Cubox** [comment]
Problem fixed.

**Comment by eirikb** [comment]
Just a note: Proper SSL support was included in 2770850cdca92735e0c338316f0f63c29f7786f8 and https://github.com/eirikb/weechat.js/commit/9108dc64a006f092d323eccd506a62671a5c5693
